docs/examples.md

```diff
@@ -323,7 +323,7 @@ that caches installed packages based on contents of `requirements.txt` and runs
 
 ```yaml
 container:
-  image: python:latest
+  image: python:slim
```

Comment:
Why not using the default image?

Reply:
Because it is smaller, meaning the tests start faster